thunderbird/processes/wps_split_merged_climos.py

```diff
@@ -0,0 +1,118 @@
+            "NOTSET",
+            "DEBUG",
+            "INFO",
+            "WARNING",
+            "ERROR",
+            "CRITICAL",
+        ]
+        inputs = [
+            ComplexInput(
+                "netcdf",
+                "NetCDF Datasets",
+                abstract="NetCDF files to process",
+                min_occurs=1,
+                max_occurs=MAX_OCCURS,
+                supported_formats=[FORMATS.NETCDF, FORMATS.DODS],
+            ),
+            LiteralInput(
+                "loglevel",
+                "Log Level",
+                default="INFO",
+                abstract="Logging level",
+                allowed_values=self.log_level_choices,
+            ),
+        ]
+        outputs = [
+            ComplexOutput(
+                "output",
+                "Output",
+                abstract="Collection of split climatological files",
+                as_reference=True,
+                supported_formats=[FORMATS.META4],
+            ),
+        ]
+
+        super(SplitMergedClimos, self).__init__(
+            self._handler,
+            identifier="split_merged_climos",
+            title="Split Merged Climos",
+            abstract="Split climo means files into one file per time interval",
+            metadata=[
+                Metadata("NetCDF processing"),
+                Metadata("Climate Data Operations"),
+            ],
+            inputs=inputs,
+            outputs=outputs,
+            store_supported=True,
+            status_supported=True,
+        )
+
+    def get_filepaths(self, request):
+        filepaths = []
+        for path in request.inputs["netcdf"]:
+            if is_opendap_url(path.url):
+                filepaths.append(path.url)
+            elif path.file.endswith(".nc"):
+                filepaths.append(path.file)
+            else:
+                raise ProcessError(
+                    "You must provide a data source (opendap/netcdf). "
+                    f"Inputs provided: {request.inputs}"
+                )
+        return filepaths
+
+    def _handler(self, request, response):
+        response.update_status("Starting Process", 0)
+        loglevel = request.inputs["loglevel"][0].data
+        logger.setLevel(getattr(logging, loglevel))
+        filepaths = self.get_filepaths(request)
+        response.update_status("Processing files", 10)
+        output_filepaths = []
+        for path in filepaths:
+            logger.info("")
+            logger.info(f"Processing: {path}")
+            try:
+                input_file = CFDataset(path)
+            except Exception as e:
+                logger.info(f"{e.__class__.__name__}: {e}")
+            else:
+                output_filepaths.extend(split_merged_climos(input_file, self.workdir))
+
+        response.update_status("File processing complete", 90)
+        response.outputs["output"].data = build_meta_link(
+            "split_climo", "Split climatologies", output_filepaths, self.workdir,
+        )
```

Comment:
This looks really good, but it may be made more clear by assigning the string params by keyword.  As this method call stands right now it is clear what the final 2 params are, but the first 2 are less clear. Something like:

```python3
response.outputs["output"].data = build_meta_link(
    varname="split_climo", desc="Split climatologies", output_filepaths, self.workdir,
)
```